client/express-checkout/blocks/components/express-checkout-component.js

```diff
@@ -40,7 +40,10 @@ const adjustButtonHeights = ( buttonOptions, expressPaymentMethod ) => {
 	// Apple Pay has a nearly imperceptible height difference. We increase it by 1px here.
 	if ( buttonOptions.buttonTheme.applePay === 'black' ) {
 		if ( expressPaymentMethod === 'applePay' ) {
-			buttonOptions.buttonHeight = buttonOptions.buttonHeight + 0.4;
+			buttonOptions.buttonHeight = Math.min(
+				buttonOptions.buttonHeight + 0.4,
+				55
+			); // Max allowed size is 55px.
```

Comment:
Keeping this line but making sure we won't exceed the maximum allowed.

Reply:
Let's put the comment above the assignment to `buttonHeight`. I find it a little odd to read it after the assignment 🤔